.gitignore

```diff
@@ -1,3 +1,5 @@
 .idea/*
 *.iml
 target/*
+.logs/*
+*.log
```

Comment:
remove these lines from here

Reply:
Then the logs will go to the github, which is a bad practice, as I understand it